From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Ed Chapman - Turing <edchapmanelc@gmail.com>
Date: Wed, 23 Oct 2024 14:22:46 +0100
Subject: [PATCH] for multi-node launching only

---
 megatron/neox_arguments/arguments.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/megatron/neox_arguments/arguments.py b/megatron/neox_arguments/arguments.py
index 15302308..b8da10af 100644
--- a/megatron/neox_arguments/arguments.py
+++ b/megatron/neox_arguments/arguments.py
@@ -405,9 +405,7 @@ class NeoXArgs(*BASE_CLASSES):
                 )
 
         # Ed: Include this arg for single node slurm or mpi launching **only**
-        args_list.extend(
-            self.convert_key_value_to_command_line_arg("force_multi", True)
-        )
+        # args_list.extend(self.convert_key_value_to_command_line_arg("force_multi", True))
 
         if "DLTS_HOSTFILE" in os.environ:
             args_list.extend(
